components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -41,17 +42,20 @@ type Props = {
     ariaLabel?: string;
     root?: boolean;
     show?: boolean;
+    direction?: 'left' | 'right';
```

Comment:
Should we make consistent with the `openUp` prop and use `openLeft` instead? Both are defining direction.

Reply:
We actually need to rework all this anyways, will discuss outside of this PR.

Reply:
Yup, no worries.